<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EMailMaaritys</class>
 <widget class="QWidget" name="EMailMaaritys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>1237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="sslVaro">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../pic/pic.qrc">:/pic/lukitsematta.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sslTeksti">
       <property name="text">
        <string>Tarkastetaan, onko SSL käytettävissä...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Sähköpostiasetusten tallennus</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="tkAsetusRadio">
        <property name="text">
         <string>Tietokoneen asetuksiin</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="kpAsetusRadio">
        <property name="text">
         <string>Kirjanpitokohtaisesti</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Smtp-palvelin</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Palvelin</string>
        </property>
        <property name="buddy">
         <cstring>palvelinEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="palvelinEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Portti</string>
        </property>
        <property name="buddy">
         <cstring>porttiSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="porttiSpin">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>65000</number>
        </property>
        <property name="value">
         <number>465</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Käyttäjänimi</string>
        </property>
        <property name="buddy">
         <cstring>kayttajaEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="kayttajaEdit"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Salasana</string>
        </property>
        <property name="buddy">
         <cstring>salasanaEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="salasanaEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="tyyppiCombo">
        <item>
         <property name="text">
          <string>Suojaamaton yhteys (TCP)</string>
         </property>
         <property name="icon">
          <iconset resource="../pic/pic.qrc">
           <normaloff>:/pic/lukitsematta.png</normaloff>:/pic/lukitsematta.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Salattu yhteys (SSL)</string>
         </property>
         <property name="icon">
          <iconset resource="../pic/pic.qrc">
           <normaloff>:/pic/lukittu.png</normaloff>:/pic/lukittu.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Suojattu yhteys (STARTTLS)</string>
         </property>
         <property name="icon">
          <iconset resource="../pic/pic.qrc">
           <normaloff>:/pic/lukittu.png</normaloff>:/pic/lukittu.png</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="testiLabel">
       <property name="text">
        <string>Testisähköpostia lähetetään...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="kokeileNappi">
       <property name="text">
        <string>Kokeile</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/email.png</normaloff>:/pic/email.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Viestien lähettäjä</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Nimi</string>
        </property>
        <property name="buddy">
         <cstring>nimiEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nimiEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Sähköpostiosoite</string>
        </property>
        <property name="buddy">
         <cstring>emailEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="emailEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Laskukopio osoitteeseen</string>
       </property>
       <property name="buddy">
        <cstring>kopioEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="kopioEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tkAsetusRadio</tabstop>
  <tabstop>kpAsetusRadio</tabstop>
  <tabstop>palvelinEdit</tabstop>
  <tabstop>porttiSpin</tabstop>
  <tabstop>tyyppiCombo</tabstop>
  <tabstop>kayttajaEdit</tabstop>
  <tabstop>salasanaEdit</tabstop>
  <tabstop>kokeileNappi</tabstop>
  <tabstop>nimiEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>kopioEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
